photon-server/src/main/java/org/photonvision/common/hardware/metrics/CPUMetrics.java

```diff
@@ -26,10 +26,13 @@ public String getMemory() {
         return execute(cpuMemoryCommand);
     }
 
-    // TODO: Command should return in Celsius
     public String getTemp() {
         if (cpuTemperatureCommand.isEmpty()) return "";
-        return execute(cpuTemperatureCommand);
+        try {
+            return String.valueOf(Double.parseDouble(execute(cpuTemperatureCommand)) / 1000);
```

Comment:
Converting the output of the command to a string without stripping away non-numeric characters somethings threw errors, would probably add some sort of regex before parsing.